### DISCUSSION TITLE
Display image url as 'clickable' thumbnail

### DISCUSSION THREAD

**Discussion by lonnylundsten** [datapane/datapane#367] [Ideas]
### Goals

1. When displaying a DataFrame that contains image url's, those url's should have the options of both being clickable and, also,  displaying a thumbnail of the image.
2.
3.

### Non-Goals

1.
2.
3.

### Background

This is available in Pandas and a useable feature in Jupyter notebooks.

### Proposal

Meerkat has an interesting method of displaying url's as thumbnails and clickable links:
http://meerkat.wiki/docs/start/quickstart-df.html

**Comment by mands** [comment]
Hi @lonnylundsten,

Thanks for your message, I think that's something we can support - we're looking at overhauling our DataTable component and can add more media support to them. Will add to the backlog!
